packages/foreman/rubygem-dynflow/rubygem-dynflow.spec

```diff
@@ -1,93 +1,107 @@
+# Generated from dynflow-1.3.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name dynflow
 
-Summary: DYNamic workFLOW engine
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.2.3
+Version: 1.3.0
 Release: 1%{?foremandist}%{?dist}
+Summary: DYNamic workFLOW engine
 Group: Development/Languages
 License: MIT
 URL: https://github.com/Dynflow/dynflow
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-%if 0%{?el6} && 0%{!?scl:1}
-Requires: %{?scl_prefix_ruby}ruby(abi)
-BuildRequires: %{?scl_prefix_ruby}ruby(abi)
-%else
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-%endif
-
+Requires: %{?scl_prefix_ruby}ruby >= 2.3.0
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ror}rubygem(multi_json)
+Requires: %{?scl_prefix}rubygem(apipie-params)
 Requires: %{?scl_prefix}rubygem(algebrick) >= 0.7.0
-Requires: %{?scl_prefix}rubygem(algebrick) < 0.8.0
+Requires: %{?scl_prefix}rubygem(algebrick) < 0.8
 Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) >= 1.1.3
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) < 1.2.0
+Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) < 1.2
 Requires: %{?scl_prefix}rubygem(concurrent-ruby-edge) >= 0.4.1
-Requires: %{?scl_prefix}rubygem(concurrent-ruby-edge) < 0.5.0
-Requires: %{?scl_prefix_ror}rubygem(multi_json)
-Requires: %{?scl_prefix}rubygem(apipie-params)
-Requires: %{?scl_prefix}rubygem-sequel >= 4.0.0
-Requires: %{?scl_prefix}rubygem-statsd-instrument
```

Comment:
Looks like this is a soft dependency in the Gemfile introduced in 8cdd0c1f64be53df7d9cbd760c1741bda3c8caf4. @iNecas should we manually maintain this dependency?

Reply:
Deferring to @adamruzicka